### PR TITLE
Russian translation of items on the Jenkins configuration page

### PR DESCRIPTION
**Pull request by Riliane**
Besides translating new lines, some outdated translations have been replaced (mentioning "tied jobs" and "master nodes"). An incorrect case letter preventing the message from actually localizing has been found in an Italian messages file (when copying it to modify to Russian 😄) and fixed. One display name has been un-hardcoded and moved to a messages bundle.

To test it view the Manage Jenkins -> System Configuration page sending a `Accept-Language:ru-RU` request header.

### Proposed changelog entries

- Add translation of the Jenkins system configuration page content into Russian

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [N/A] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [N/A] There is automated testing or an explanation as to why this change has no tests.
- [N/A] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [N/A] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [N/A] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [N/A] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [N/A] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@Pldi23 @yaroslavafenkin @duemir 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7272"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/model/AbstractItem/noWorkspace_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/model/AbstractItem/noWorkspace_ru.properties:25]:
```suggestion
Проект не будет иметь рабочую директорию, пока хотя бы одна сборка не будет запущена.
```

File: core/src/main/resources/hudson/model/Messages_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/model/Messages_ru.properties:88]: Hm, looks weird, Node.Mode.NORMAL but description as it is Node.Mode.HIGH or equal.

File: core/src/main/resources/hudson/model/UpdateCenter/CoreUpdateMonitor/description_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/model/UpdateCenter/CoreUpdateMonitor/description_ru.properties:1]:
```suggestion
blurb = При выходе новой версии Jenkins это сообщение проинформирует о ней администраторов.
```

File: core/src/main/resources/hudson/slaves/Messages_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/slaves/Messages_ru.properties:25]:
```suggestion
RetentionStrategy.Demand.OfflineIdle=Выключен, т.к. компьютер бездействовал; будет перезапущен, когда потребуется
```

File: core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/description_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/description_ru.properties:1]:
```suggestion
blurb = Предупреждает администраторов, если триггеры не срабатывают по расписанию - обычно потому, что \
```

File: core/src/main/resources/hudson/triggers/SCMTrigger/AdministrativeMonitorImpl/description_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/triggers/SCMTrigger/AdministrativeMonitorImpl/description_ru.properties:1]:
```suggestion
blurb = Если в расписании больше опросов SCM, чем Jenkins может выполнить параллельно, это предупреждение информирует \
```

File: core/src/main/resources/jenkins/model/BuiltInNodeMigration/description_ru.properties
Comment (Pldi23) [inline core/src/main/resources/jenkins/model/BuiltInNodeMigration/description_ru.properties:1]:
```suggestion
blurb = Позволяет администраторам мигрировать имя и метку встроенного узла при обновлении с версии Jenkins старше \
```

**Review by Pldi23 — COMMENTED** [review]
Nicely done! left few not so important suggestions

File: core/src/main/resources/hudson/model/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/model/Messages_ru.properties:88]: the options are NORMAL and EXCLUSIVE, so I think it's "normal" as in "default", "as usual".

File: core/src/main/resources/hudson/model/AbstractItem/noWorkspace_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/model/AbstractItem/noWorkspace_ru.properties:25]:
```AbstractProject.NoWorkspace=Сборочной директории нет, не могу проверить обновления.
AbstractProject.WorkspaceTitle=Сборочная директория {0}
AbstractProject.WorkspaceTitleOnComputer=Сборочная директория {0} на {1}```
maybe in that case we should unify that too. I do remember it being "рабочая директория" in the job config.

Comment (Riliane) [reply core/src/main/resources/hudson/model/AbstractItem/noWorkspace_ru.properties]: actually, ran a search, in a lot of places it's сборочная директория, in a few (including mine) - рабочая директория. either way we should pick one and stick to it

Comment (Pldi23) [reply core/src/main/resources/hudson/model/AbstractItem/noWorkspace_ru.properties]: ok, that means most of users already got used to "сборочная директория", fine for me then

File: core/src/main/resources/hudson/model/Messages_ru.properties
Comment (Pldi23) [reply core/src/main/resources/hudson/model/Messages_ru.properties:88]: thanks for explanation

**Review by Pldi23 — APPROVED** [review]

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/Messages_ru.properties:26]: "войдите в систему как этот пользователь" sounds very unnatural. Not yet sure how to word it better. Also missing context.

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/Messages_ru.properties:44]: Not a big fan of "экземпляр". Maybe "на другую установку" / "на другую инстанцию"? Although these two are also odd it reads a bit better than the current version to me. Or maybe just "на другой Jenkins"?

File: core/src/main/resources/hudson/diagnosis/ReverseProxySetupMonitor/description_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/diagnosis/ReverseProxySetupMonitor/description_ru.properties:1]: it was left untranslated in what was there before me, so I left it like that to match - I'll consider that and look if there's more untranslated usages

File: core/src/main/resources/jenkins/diagnostics/RootUrlNotSetMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/diagnostics/RootUrlNotSetMonitor/message_ru.properties:3]: "некорректным" is a good idea for "invalid", thanks

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:26]:
```// you are the user or you have ADMINISTER permission
            u.checkPermission(Jenkins.ADMINISTER);
...
...
return HttpResponses.html(p.hasPermissionToSeeToken()
                    ? Messages.ApiTokenProperty_ChangeToken_Success()
                    : Messages.ApiTokenProperty_ChangeToken_SuccessHidden());
```
Looks like it's either you yourself or an admin that can generate this token for you, but only you can see it.If an admin generated this token, they get this message instead of not seeing it.
```
<div>Updated. You need to login as the user to see the token</div>
```

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:29]: good point. "не имея уже существующего" sounds better?

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:44]: I have considered just skipping the word "instance" and just using "на другой Jenkins", but there's a line where I couldn't avoid that and had to pick a word:
```
ResourceDomainConfiguration.FailedIdentityCheck=An error occurred when checking the instance identity at that URL: {0} {1}

ResourceDomainConfiguration.FailedIdentityCheck=Произошла ошибка при сверке экземпляра Jenkins по URL: {0} {1}
```

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:26]: Then I'd consider "войдите используя учетную запись этого пользователя" or something close to that.

File: core/src/main/resources/hudson/slaves/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/slaves/Messages_ru.properties:24]: I was asking about animate vs inanimate "agent", I've had suggestions that it's a machine with very little autonomy. I left it inanimate for now, but still considering

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/jenkins/security/Messages_ru.properties]: Yup, sounds good for me

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (Pldi23) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:44]: My 2 cents here, "экземпляр" is better then "установка" или "инстанция", but yes, sounds not natural. May be just use Jenkins without direct translation of instance would be fine...

Comment (Pldi23) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:44]: By the way, "экземпляр" sounds not natural, but very clearly describe the meaning IMO

File: core/src/main/resources/hudson/slaves/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/slaves/Messages_ru.properties:24]: considering putting "online" as "подключить/подключён" everywhere and "offline" as "отключить/отключён", but leaving "agent" inanimate for now...

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (duemir) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:44]: In the case of Jenkins, "instance" can quite literally mean "class instance" which I believe is "экземпляр" (at least Yandex Translate convinced me about it with quotes from technet.microsoft.com)

**Review by yaroslavafenkin — APPROVED** [review]

File: core/src/main/resources/jenkins/security/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/Messages_ru.properties:44]: Then I'll leave it as is.

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Review by basil — APPROVED** [review]
Thanks for the PR!

**Pull request merged**
